definitions/artifacts/aws-iam-role.json

```diff
@@ -14,6 +14,10 @@
     "containerRepositories": {
       "label": "ECR",
       "cloud": "AWS"
+    },
+    "alarmChannels": {
+      "label": "AWS",
+      "cloud": "aws"
```

Comment:
we are going to unify around all lowercase for the cloud value here. On the MD side we will make this an enum field in GQL.

Reply:
dig it.

Reply:
The way it is handled in the backend will make this tough, however, when I update container repos for synchronous communication with satellite I can fix this because it will be a BC change.